go/src/koding/klient/app/klient.go

```diff
@@ -590,6 +632,45 @@ func (k *Klient) tunnelOptions() (*tunnel.Options, error) {
 	return opts, nil
 }
 
+func (k *Klient) Team() (*stack.Team, error) {
+	var resp stack.WhoamiResponse
+
+	if err := k.kloud.Call("team.whoami", nil, &resp); err != nil {
+		return nil, err
+	}
+
+	return resp.Team, nil
+}
+
+func (k *Klient) cacheTeam() (*stack.Team, error) {
+	k.teamMu.Lock()
+	defer k.teamMu.Unlock()
+
+	if k.team != nil {
+		return k.team, nil
+	}
+
+	team, err := k.Team()
```

Comment:
@ppknap This will fail for KD Daemon due to https://github.com/koding/koding/issues/8514 - klient for KD does not exist in jMachines currently. You should not block the klient if Team() fails and env ~= *managed.